code/mob/living.dm

```diff
@@ -181,6 +182,8 @@
 /mob/living/death(gibbed)
 	#define VALID_MOB(M) (!isVRghost(M) && !isghostcritter(M) && !inafterlife(M))
 	src.remove_ailments()
+	if (src.faction)
+		src.faction = null
```

Comment:
will cause SRed/otherwise-revived mobs to be factionless. Intentional?

Reply:
No, was intended to prevent respawned mobs from keeping factions across mobs, but I believe the mobs faction is set on creation, so I'll remove this.